### DISCUSSION TITLE
Hi,  adjacent word repition in rg.

### DISCUSSION THREAD

**Discussion by jianhe-fun** [BurntSushi/ripgrep#2140] [Q&A] [answered]
I tried. then I failed. 
https://blog.burntsushi.net/ripgrep/
How to find add adjacent word repetition in rg.
for example string as: _"hello hello world"_
how to found out _"hello hello"_.

**Comment by BurntSushi** [comment, marked as answer]
```
$ echo 'hello hello world' | rg -Po '(\w+) (\1)'
hello hello
```
